src/styles/card.css

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
```

Comment:
We are using this color a lot in this block, let's move it to the variables

Reply:
done